data/search_synonyms.json

```
@@ -13,10 +13,89 @@
     "HS": ["Hörsaal", "Vorlesungssaal"],
     "Hörsaal": ["HS", "Vorlesungssaal"],
     "Seminarraum": ["Übungsraum"],
-    "Lernraum": ["Seminarraum", "Studentenarbeitsraum", "Übungsraum"],
+    "Lernraum": ["Lernplatz", "Seminarraum", "Studentenarbeitsraum", "Übungsraum"],
     "Übungsraum": ["Seminarraum", "Studentenarbeitsraum", "Übungsraum"],
-    "Arbeitsraum": ["Seminarraum", "Studentenarbeitsraum", "Übungsraum"],
+    "Arbeitsraum": ["Seminarraum", "Studentenarbeitsraum", "Übungsraum"], 
     
+    "Rechnerraum-Studenten": ["Rechnerarbeitsraum", "Rechnerraum", "Rechner-Raum", "Computerarbeitsraum", "Computerraum", "Computerpool", "Computer Studenten", "CIP-Raum", "CIP-Pool", "Computerpool", "CAD-Pool", "EDV-Pool"],
+    "Rechnerarbeitsraum": ["Rechnerraum-Studenten", "Rechnerraum", "Rechner-Raum", "Computerarbeitsraum", "Computerraum", "Computerpool", "Computer Studenten", "CIP-Raum", "CIP-Pool", "Computerpool", "CAD-Pool", "EDV-Pool"],
+    "Rechnerraum": ["Rechnerraum-Studenten", "Rechnerarbeitsraum", "Rechner-Raum", "Computerarbeitsraum", "Computerraum", "Computerpool", "Computer Studenten", "CIP-Raum", "CIP-Pool", "Computerpool", "CAD-Pool", "EDV-Pool"],
+    "Rechner-Raum": ["Rechnerraum-Studenten", "Rechnerarbeitsraum", "Rechnerraum", "Computerarbeitsraum", "Computerraum", "Computerpool", "Computer Studenten", "CIP-Raum", "CIP-Pool", "Computerpool", "CAD-Pool", "EDV-Pool"],
```

```suggestion
    "Rechner-Raum": ["Rechnerraum-Studenten", "Rechnerarbeitsraum", "Rechnerraum", "Computerarbeitsraum", "Computerraum", "Computerpool", "Computer Studenten", "CIP-Raum", "CIP-Pool", "Computerpool", "CAD-Pool", "EDV-Pool"],
    "Rechnerhalle": ["Rechnerraum-Studenten", "Rechnerarbeitsraum", "Rechnerraum", "Computerarbeitsraum", "Computerraum", "Computerpool", "Computer Studenten", "CIP-Raum", "CIP-Pool", "Computerpool", "CAD-Pool", "EDV-Pool", "Rechner-Raum"],
```

"Rechnerhalle" is nowhere in the data, but might also make sense as a search keyword